modal/_container_entrypoint.py

```diff
@@ -149,6 +168,9 @@ async def get_serialized_function(self) -> tuple[Optional[Any], Callable]:
 
         return cls, fun
 
+    def exc_info(self) -> ExceptionInfo:
+        return self._exc_info
```

Comment:
I don't really understand why this needs to be a method — I assume it's some synchronicity thing, but when I tried just populating an attribute on the `FunctionIOManager` class it didn't work, the client code couldn't see that the attribute existed.